hack/make/.build-deb/control

```diff
@@ -15,9 +15,9 @@ Recommends: aufs-tools,
             ${apparmor:Recommends}
 Conflicts: docker (<< 1.5~), docker.io, lxc-docker, lxc-docker-virtual-package
 Description: Docker: the open-source application container engine
- Docker is an open source project to pack, ship and run any application as a
- lightweight container
- .
+ Docker is an open source project to build, ship and run any application as a
+ lightweight container.
+
```

Comment:
The period on this blank line is actually important for how this file is parsed; can we get that added back please? :+1: